app/src/main/java/com/example/cinemapp/AppModule.kt

```diff
@@ -46,19 +51,24 @@ val appModule = module {
     single {
         MovieRepository(get(), get())
     }
+    single { URLMapper() }
```

Comment:
Camel case for word `URL` would be `Url`. So -> `UrlMapper`...And maybe even specifically, what URL are you trying to map? It is a bit generic, don't you think? 😅

Reply:
Well the current UrlMapper maps TMDB image ids and Youtube video ids so maybe MediaUrlMapper would work? Or should i split it into 2 mappers?

Reply:
That should work for now :D